hardware_interface/include/hardware_interface/robot_hw.h

```diff
@@ -148,33 +178,59 @@ class RobotHW : public InterfaceManager
     ERROR
   };
 
-  // Return (in realtime) the state of the last doSwitch()
+  /** \brief Return (in realtime) the state of the last doSwitch() */
   virtual SwitchState switchResult() const
   {
     return DONE;
   }
 
-  // Return (in realtime) the state of the last doSwitch() for a given controller
+  /** \brief Return (in realtime) the state of the last doSwitch() for a given controller */
   virtual SwitchState switchResult(const ControllerInfo& /*controller*/) const
   {
     return DONE;
   }
+  /**\}*/
 
-  /**
-   * Reads data from the robot HW
+  /** \name Control Loop
+   *\{*/
```

Comment:
I don't know how to make these groups look alike. `Hardware Interface Switching` seems to be more important to doxygen than `Resource Management` and `Control Loop`.

Reply:
See https://github.com/ros-controls/ros_control/pull/433#discussion_r405916399